src/backend/common/half.hpp

```diff
@@ -77,7 +76,7 @@ CONSTEXPR_DH uint16_t float2half(float value) {
     uint32_t bits = 0;  // = *reinterpret_cast<uint32*>(&value);
                         // //violating strict aliasing!
     std::memcpy(&bits, &value, sizeof(float));
-    constexpr uint16_t base_table[512] = {
+    uint16_t base_table[512] = {
```

Comment:
Hmm. I should make that a static const array. The reason I removed constexpr is because of the memcpy later in the function. The memcpy is not a constexpr operation so the constexpr decorator would have been deceptive here. Also one of the compilers threw an error.

Reply:
Got it. Then `static const` should do the trick I guess.